README.md

```diff
@@ -67,7 +67,8 @@ If you want to customize the application environment, reference this table:
 | WALLET_SERVICE_URI                                   | Wallet service endpoint                                                                    | string |                         |
 | WALLET_SERVICE_READ_TIMEOUT                          | Wallet service read timeout                                                                | number | 10000                   |
 | WALLET_SERVICE_CONNECTION_TIMEOUT                    | Wallet service connection timeout                                                          | number | 10000                   |
```

Comment:
Add newly added CDC_SEND_RETRY_MAX_ATTEMPTS and CDC_SEND_RETRY_INTERVAL_IN_MS

Reply:
ops. fixed here 381a8d0f3a2881bb1e7e2b199b50204433927b5c